community/fzf/APKBUILD

```diff
@@ -41,19 +47,38 @@ package() {
 
 tmux() {
 	depends="tmux bash"
-	description="Helper script to start fzf in a tmux pane"
+	pkgdesc="Helper script to start fzf in a tmux pane"
 
 	cd "$builddir"
 	install -Dm0755 bin/fzf-tmux "$subpkgdir"/usr/bin/fzf-tmux
 }
 
-extras() {
-        description="additional scripts like shell completion and keybindings"
+bashcomp() {
+	pkgdesc="additional scripts for bash like shell completion and keybindings"
+	install_if="$pkgname=$pkgver-r$pkgrel bash-completion"
 
-        cd "$builddir"
-        install -dm0755 "$subpkgdir"/usr/share/fzf
-        install -m0644 shell/* "$subpkgdir"/usr/share/fzf
+	cd "$builddir"
+	install -Dm0644 shell/completion.bash "$subpkgdir"/usr/share/bash-completion/completions/$pkgname
+	install -Dm0644 shell/key-bindings.bash "$subpkgdir"/usr/share/$pkgname/key-bindings.bash
+}
+
+zshcomp() {
+	pkgdesc="additional scripts for zsf like shell completion and keybindings"
+	install_if="$pkgname=$pkgver-r$pkgrel zsh"
+
+	cd "$builddir"
+	install -Dm0644 shell/completion.zsh "$subpkgdir"/usr/share/zsh/site-functions/_$pkgname
+	install -Dm0644 shell/key-bindings.zsh "$subpkgdir"/usr/share/$pkgname/key-bindings.zsh
+}
+
+fishcomp() {
+	pkgdesc="keybindings for fish"
+	install_if="$pkgname=$pkgver-r$pkgrel fish"
+
+	cd "$builddir"
+	install -Dm0644 shell/key-bindings.zsh "$subpkgdir"/usr/share/fzf/keybindings.fish
```

Comment:
I wouldn't ship the keybinding in a completion package since it is not a shell completion :wink: I would instead suggest installing all keybindgs from the `shell/` directory to `/usr/share/doc/$pkgname/examples` or something like that and make them part of the -doc subpackage.